src/utils/swapMinAmounts.ts

```diff
@@ -42,27 +42,69 @@ export const createDisabledSwapPluginMap = (
 export async function swapMinAmounts(
   account: EdgeAccount,
   currencyWallets: EdgeCurrencyWallet[]
-): Promise<EdgeSwapQuote[]> {
+): Promise<{ [key: string]: number }> {
+  // Helper function to find minimum swap amount
+  const findMinSwapAmount = async (
+    min: number,
+    max: number,
+    fromWallet: EdgeCurrencyWallet,
+    toWallet: EdgeCurrencyWallet,
+    reqOptions: EdgeSwapRequestOptions
+  ): Promise<number> => {
+    // Eventually max is going to be below the minimum amount by 1
+    if (min === max + 1) {
+      return min
+    }
```

Comment:
This can be a single line

Reply:
Needed to add a variable inside the `if` block for the `multiplier` to return the minimum amount with the right decimal places. Reviewed changes with Lex.